lms/product/moodle/_plugin/course_copy.py

```diff
@@ -31,6 +31,55 @@ def find_matching_group_set_in_course(self, course, group_set_id):
             course, group_set_id
         )
 
+    def effective_document_id(
+        self,
+        log,
+        document_url,
+        document_course_id,
+        document_file_id,
+        course,
+        current_course_id,
```

Comment:
current_course_id could be course.lms_id but I was thinking ahead for Canva's version where that's not the case.

This is Moodle's only file so we could do that and save one parameter.